css/stylesheet.css

```diff
@@ -1,6 +1,7 @@
 *{
   box-sizing: border-box;
   font-family:fantasy, 'Arial Narrow Bold', sans-serif;
+  color: white;
```

Comment:
If you add white color here i think your text or the content of your page will not shown as need. so make sure your output is good.